saml/saml.php

```diff
@@ -11,7 +11,6 @@
 use Friendica\Core\Hook;
 use Friendica\Core\Logger;
 use Friendica\Core\Renderer;
-use Friendica\Core\Session;
```

Comment:
There are references to `$_SESSION` in this file that should be replaced by `Core\Session` calls.